### DISCUSSION TITLE
AutoCad freezes when editing text in paper space

### DISCUSSION THREAD

**Discussion by pvk988** [mozman/ezdxf#754] [Issues]
Hello. AutoCad freezes when editing text or mtext in paper space
[test_VIEWPORTS.zip](https://github.com/mozman/ezdxf/files/9786141/test_VIEWPORTS.zip)

**Comment by pvk988** [comment]
after editing the block, the problem disappears

**Comment by mozman** [comment]
No issues editing the text in BricsCAD and I don't use AutoCAD, so I can't help without a useful error message.
